remotespark/utils/sparkevents.py

```diff
@@ -22,10 +22,10 @@ def emit_session_creation_start_event(self, session_guid, language):
         assert language in constants.SESSION_KINDS_SUPPORTED
 
         event_name = constants.SESSION_CREATION_START_EVENT
-        time_stamp = SparkEvents.get_utc_date_time()
+        time_stamp = "{}".format(SparkEvents.get_utc_date_time())
 
-        kwargs_list = [(constants.EVENT_NAME, event_name), (constants.TIMESTAMP, time_stamp), (constants.SESSION_GUID, session_guid),
-                       (constants.LIVY_KIND, language)]
+        kwargs_list = [(constants.EVENT_NAME, event_name), (constants.TIMESTAMP, time_stamp),
+                       (constants.SESSION_GUID, "{}".format(session_guid)), (constants.LIVY_KIND, language)]
```

Comment:
No, it's better to use the `str()` function since you're not actually doing any string formatting